### PR TITLE
Add the final Trino Summit teaser blog

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-10-31-trino-summit-2022-teaser-3.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+<p align="center">
+<img align="center" width="75%" src="/assets/blog/trino-summit-2022/san-francisco.png"/><br/>
+</p>
+
+We know this only applies to our in-person attendees, but remember if you join
+us virtually, there are still plenty of resources to network and interact 
+throughout the conference. We will be taking questions from our virtual audience
+and there will also be a chat forum to discuss with attendees from across the
+globe. Plus, unlike those of us attending in-person, no travel required and
```
Comment (mosabua) [inline _posts/2022-10-31-trino-summit-2022-teaser-3.md:130]:
```suggestion
globe. Plus, unlike those of us attending in-person, no travel is required and
```

File: _posts/2022-10-31-trino-summit-2022-teaser-3.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+pants are optional during the event!
```
Comment (mosabua) [inline _posts/2022-10-31-trino-summit-2022-teaser-3.md:131]: Not sure if this is okay ... maybe drop to be on the safe side

File: _posts/2022-10-31-trino-summit-2022-teaser-3.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+### 5. Collaborate with some of the best minds working on Trino
```
Comment (mosabua) [inline _posts/2022-10-31-trino-summit-2022-teaser-3.md:133]: empty line after title

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-10-31-trino-summit-2022-teaser-3.md (hunk 2 above)
Comment (bitsondatadev) [reply _posts/2022-10-31-trino-summit-2022-teaser-3.md:131]: I'll change it to pjs are optional...

**Review by mosabua — APPROVED** [review]
Looks good now.

**Pull request merged**